src/AtlasAdmin.ts

```diff
@@ -0,0 +1,173 @@
+// @ts-ignore
+import Logger from 'basic-logger';
+import { request, RequestOptions } from 'urllib';
+import { SearchIndex } from './data/atlas-search-index';
+import { IndexMappings } from './data/atlas-types';
+import { Taxonomy } from './SearchIndex';
+
+const DEFAULT_ATLAS_API_OPTIONS: RequestOptions = {
+  headers: {
+    'content-type': 'application/json',
+  },
+  dataType: 'json',
+};
+
+const log = new Logger({
+  showTimestamp: true,
+});
+
+const CLUSTER_NAME = process.env['CLUSTER_NAME'] || 'Search';
+const COLLECTION_NAME = process.env['COLLECTION_NAME'] || 'documents';
+const GROUP_ID = process.env['GROUP_ID'] || '';
```

Comment:
included a `verifyEnvVars` fn to verify all the new env vars :) left out TAXONOMY_URL since it is not available yet, hence the fallback data

Reply:
Gotcha! Since the `verifyEnvVars` function returns the env variables, could we use those to access env vars?  Since it gets the env vars, maybe we could name it `verifyAndGetEnvVars`. That way, TypeScript won't complain as much

Reply:
done. passed to AtlasAdmin Manager from index.ts after getting env vars